webhook-integrations/Facebook/notes.md

```diff
@@ -0,0 +1,36 @@
+# Facebook Integration Investigation
+
+[Facebook](https://www.developers.facebook.com) is a social networking service that also allows developers
+to track analytics of their apps.
```

Comment:
Is it only for apps, or can we also receive user analytics?

Reply:
What do you mean by we?
It is for apps & websites

Reply:
Also, it is meant only for the developers, no user can view these analytics

Reply:
Okay. I was asking if it is also possible to receive analytical data for some Facebook user, e.g. receive a notification whenever user x posts a status update.

Reply:
no, should I mention that in the notes?

Reply:
@MadElf1337: Probably mention that in the notes yes. :)